optics-agent/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
```

Comment:
Use `npm run new-package -- optics-agent` to generate this file and `tslint.json`. (You'll have to rename the folder to `optics-agent-old`, the move your `index.d.ts` and `optics-agent-tests.ts` back.)